reana_workflow_controller/workflow_run_manager.py

```diff
@@ -438,7 +438,10 @@ def _create_job_spec(
 
         workflow_metadata = client.V1ObjectMeta(
             name=name,
-            labels={"reana_workflow_mode": "batch"},
+            labels={
+                "reana_workflow_mode": "batch",
+                "reana-run-batch-workflow-uuid": str(self.workflow.id_),
```

Comment:
Should we align on the naming [as here](https://github.com/reanahub/reana-job-controller/commit/9600b86e633d879a88e2a6c653dee248467562d4#diff-91c1d8d981e6114fea16cab4d9162666f97d596d1bcfc293f5a0b7e840b127b2R150) (probably your name is more complete) or do you think it might interfere? Maybe it's a good idea to keep `run-batch` and `run-jobs` pods with different label names?

Reply:
As we discussed in the standup, it's better to keep label keys different for `run-batch` and `run-jobs` pods for more precise pod selection. Perhaps we could rename `run-jobs` label keys accordingly: `reana-run-job-workflow-uuid`? WDYT? I could create a quick PR to change it

Reply:
`reana-run-job-workflow-uuid` looks good to me.

Reply:
Addressed it [here](https://github.com/reanahub/reana-job-controller/pull/340)